readme.md

```diff
@@ -42,44 +42,48 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Your Answer:**
 
----
+## mongodb is actual no-SQL database that we can store and retrieve our data to/from it and mongoose is just a library and tool that we use which makes communication with database much easier.
 
 - [ ] MongoDB uses a number of terms that may be to new to you such as database, collection, and document.
 
 * **Question:** What is the difference between the above terms? [This page](https://docs.mongodb.com/manual/core/databases-and-collections/) may help.
 
 * **Your Answer:**
 
----
+## Database holds the collection of documents.
+
+## Collection is the group of individual documents.
+
+## Document is a javascript object which holds bunch of key:value pairs.
```

Comment:
_Technically_ it's BSON data. You're right though, it essentially acts like JSON/JavaScript.